packages/salesforcedx-vscode-core/test/jest/commands/util/commandEventDispatcher.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+import { RefreshSObjectsExecutor } from '../../../../src/commands';
+import { CommandEventDispatcher } from '../../../../src/commands/util/commandEventDispatcher';
+
+describe('CommandEventDispatcher', () => {
+  describe('getInstance', () => {
```

Comment:
NIT: Since it is a method of singleton, I would also suggest another test to verify it returns a singleton